roles/engineblock/templates/engineblock.ini.j2

```diff
@@ -87,6 +87,9 @@ trustedProxyIps[] = {{ engine_trusted_proxy_ip }}
 
 base_domain = {{ engine_base_domain }}
 
+; the value for guest qualifier. Can be overridden for specific non-surfnet environments
```

Comment:
Would remove 'non-surfnet' since there's not anything surf to be seen in the value now.